src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/ConvertToLibraryImportAnalyzerTests.cs

```diff
@@ -35,7 +35,6 @@ public class ConvertToLibraryImportAnalyzerTests
             new object[] { typeof(int*) },
             new object[] { typeof(bool*) },
             new object[] { typeof(char*) },
-            new object[] { typeof(delegate* <void>) },
```

Comment:
Was this causing a failure? I see that it degrades to an `IntPtr`, but the validation is still important for the Interop team. If it is causing a failure, please comment it out and create an issue for us to address when these changes go in.

Reply:
Was - I didn't think the test was relevant any more. I added it back, commented out, along with a link to the issue https://github.com/dotnet/runtime/issues/71891